include/wx/private/spinctrl.h

```diff
@@ -10,6 +10,11 @@
 #ifndef _WX_PRIVATE_SPINCTRL_H_
 #define _WX_PRIVATE_SPINCTRL_H_
 
+#ifdef __WXOSX_IPHONE__
+#include "wx/gdicmn.h"
+#include "wx/control.h"
+#endif
+
```

Comment:
I think this file is not supposed to be used at all under iOS because there is no `wxSpinCtrl` there AFAICS. So I don't really understand why is this necessary. Could you please explain what was the error which required this change?

Reply:
this was also not my preferred solution . 
the problem was 
In file included from ../src/common/spinctrlcmn.cpp:28:
../include/wx/private/spinctrl.h:32:8: error: unknown type name 'wxSize'
extern wxSize GetBestSize(const wxControl* spin, int minVal, int maxVal, int base);
       ^
../include/wx/private/spinctrl.h:32:33: error: unknown type name 'wxControl'
extern wxSize GetBestSize(const wxControl* spin, int minVal, int maxVal, int base);

it seems, that these both header files are not included in the build path.

Reply:
This one is simple to fix so I just did this blindly in 6c061b6bf91e8cd825c28e0af9875431a3e6d7b0